engine/src/main/java/org/hibernate/search/bridge/util/impl/NumericFieldUtils.java

```diff
@@ -24,6 +41,9 @@
 
 	private static final Log log = LoggerFactory.make();
 
+	private static final Class<?>[] NUMERIC_JAVA_TIME_TYPES = { Year.class, Instant.class, LocalDate.class, LocalDateTime.class, YearMonth.class,
```

Comment:
it's not initialized when you call it but when the class is initialized: i.e. whenever any code calls any method of _NumericFieldUtils_ .

> I didn't have error when building with JDK7.

How do you build with JDK7?